vipps-plugin-development.md

```diff
@@ -0,0 +1,85 @@
+
```

Comment:
we need to add some metadata to the top here. I need to tweak the docusaurus sidebar.js file too. I can test it out to see what metadata we need here. :-)

Reply:
Probably this:

```
<!-- START_METADATA
---
title: Checklist
pagination_next: null
pagination_prev: null
---
END_METADATA -->
```